### ISSUE TITLE
Release BotKube 0.14

### ISSUE THREAD

**Issue by pkosiec** [kubeshop/botkube#741]
## Overview

This task aggregates all the to-dos which should be done when it comes to 0.14 release.

## AC
- [x] Go through the release process instruction and release 0.14
    - Update it if it is outdated
    - Blocked by #584  
- [x] Describe any breaking changes in the GitHub release
- [x] Testing
    - Test manually interactivity for Slack
    - Test telemetry
    - Aggregate all stuff for manual testing described by @mszostok in https://github.com/kubeshop/botkube/issues/595
- [x] Schedule all community activities - blog post, sharing the news in social media, livestream, etc. (@brampling)

**Comment by mszostok** [comment]
Announce that you can also prefix your commands with `kubectl` , `kc` or `k`. In future, one of the kubectl prefix (`kubectl` , `kc` or `k`) will be required.

**Comment by pkosiec** [comment]
**Breaking change:**
- The `settings.configWatcher` option has been replaced with `configWatcher.enabled: true`. Config Watcher now runs as a sidecar container. See possible configuration options on the [Helm chart parameters](https://botkube.io/docs/next/configuration/helm-chart-parameters) page.

**Comment by mszostok** [comment]
**Bugs:**

- Remove the `:crossed_fingers:` form stop message, see: https://github.com/kubeshop/botkube/blob/46129cf1c2c95118a3c178bb6ea85649d6289c1c/pkg/controller/controller.go#L29
   - Problem: it's not properly formated on Discord (they have `:fingers_crossed:` 🙄 )
       ![Screen Shot 2022-09-30 at 12 57 26](https://user-images.githubusercontent.com/17568639/193255544-88b86f61-7306-440d-9de4-cff5a057b302.png)

**Comment by huseyinbabal** [comment]
## Release activities

- ✅ Ensure that all security vulnerabilities are fixed or mitigated                                                                                       
  - assignee: 

- ✅ Cut initial release candidate  for`v0.14.0`                                                                                                        
  - assignee: @huseyinbabal                                                                                                    

- ⏳ Finalize release the `v0.14.0`                                                                                                       
  - assignee: @huseyinbabal 

- ⏳  Test Slack         
    - assignee: ❓ 

- ⏳  Test Mattermost         
    - assignee: ❓ 
 
- ⏳  Test Discord         
    - assignee: ❓ 
 
- ⏳  Test MS Teams         
    - assignee: ❓ 
    
- ⏳  Test Elasticsearch         
    - assignee: ❓ 

- ⏳  Test Wbhook         
    - assignee: ❓ 

- ⏳  Check if we described all breaking changes in the release summary and if it's rendered correctly. 
  - assignee: team

**Descoped:**
- Prepare the release notes                                                                                                                             
  - assignee: @brampling 
- Create a blog post and cross-post it (e.g. Medium, Dev.to)                                                                                            
  - assignee: @brampling 
- Share the news on social media (e.g. Reddit, Hacker News, LinkedIn, Twitter, Slack)                                                                   
  - assignee: @brampling

**Comment by pkosiec** [comment]
Closing as completed 👍 Thank you for the effort!
